trp_use_case/model/use_case.py

```diff
@@ -59,18 +59,19 @@ def _get_use_case_hours(self):
             values = {
                 'hours': 0.0,
                 'hours_optional': 0.0,
+                'hours_nonoptional': 0.0,
```

Comment:
this one is not strictly necessary because the field is recomputed entirely below. But also doesn't hurt.